samples/unit-tests/rangeselector/dropdown/demo.js

```diff
@@ -43,9 +33,10 @@ QUnit.test('RangeSelector.dropdown', assert => {
         chart.rangeSelector.buttons.every(b => b.visibility !== 'hidden'),
         '800px + responsive: All the buttons should be visible'
     );
-    assert.notOk(
-        isSelectInsideChart(),
-        'Select should be outside chart'
+    assert.ok(
+        chart.rangeSelector.dropdown.style.visibility === 'hidden' &&
+        chart.rangeSelector.dropdownLabel.visibility === 'hidden',
+        'Dropdown and its label should be hidden'
```

Comment:
Could we change it to `assert.equals`? (and possibly just check one property)

Reply:
As we agreed, separating these two into two separate asserts (since they check two different elements)